airbyte-config/init/src/main/resources/config/STANDARD_SOURCE_DEFINITION/11856c6d-37c3-4cdf-81c0-f1fe2fbaf280.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sourceDefinitionId": "11856c6d-37c3-4cdf-81c0-f1fe2fbaf280",
+  "name": "Infinicept",
+  "dockerRepository": "airbyte/source-infinicept",
+  "dockerImageTag": "0.1.0",
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/infinicept"
+}
```

Comment:
You don't need to add this file anymore. Only add the same info inside the source_specs.json file.